docs/03-language-guide/14-equality.md

```diff
@@ -1,6 +1,131 @@
 ---
 title: Equality
 id: equality
+description: A reference guide to equality in Wing.
+keywords: [Winglang equality, Wing equality, equals, identity]
 ---
 
-TODO (for now, see the [language spec RFC](../999-contributors/999-rfcs/2023-06-12-language-spec.md))
+Checking for equality is performed with the `==` operator in Wing. It returns `true` if the two values are equal, and `false` otherwise.
```

Comment:
nit: I feel like our reference docs should avoid mentioning "in Wing" unless it's ambiguous (e.g. when comparing to other languages)

```suggestion
Checking for equality is performed with the `==` operator. It returns `true` if the two values are equal, and `false` otherwise.
```

Contextually, all of these docs are referring to the world of Wing so it's redundant.

Reply:
I agree